spring-cloud-build/pom.xml

```diff
@@ -125,6 +128,13 @@ limitations under the License.
 					<artifactId>asciidoctor-maven-plugin</artifactId>
 					<version>1.5.2</version>
 				</plugin>
+				<plugin>
+					<groupId>org.apache.maven.plugins</groupId>
+					<artifactId>maven-surefire-plugin</artifactId>
+					<configuration>
+						<redirectTestOutputToFile>true</redirectTestOutputToFile>
```

Comment:
> We put -Dmaven.test.redirectTestOutputToFile=true in our CI config.

I don't get it. You said earlier having it set to `true` is problematic in CI, since one can't find the files.

In any case, the rationale for my change is this: on the command line, building projects (_e.g._ s-c-data et al) is really noisy, with stacktraces popping all over the place (_e.g._ expected exceptions in tests), so I'd rather have it set to non-noisy. Any solution that allows this while not hindering CI failure analysis suits me